daphne/src/vdaf/prio2.rs

```diff
@@ -23,7 +23,7 @@ pub(crate) fn prio2_shard(
     measurement: DapMeasurement,
 ) -> Result<Vec<Vec<u8>>, VdafError> {
     let vdaf = Prio2::new(dimension as usize)?;
-    let input_shares = match measurement {
+    let (_public_share, input_shares) = match measurement {
```

Comment:
Why do we not need this?

Reply:
The public share is meaningful to Poplar1, but not Prio3. However, The VDAF API in libprio is designed to encompass all VDAFs, hence the extra output parameter that we just ignore here.